test/recipes/90-test_store.t

```diff
@@ -152,6 +150,55 @@ indir "store_$$" => sub {
                 ok(run(app(["openssl", "storeutl", to_abs_file_uri($dir, 1)])));
             }
         }
+
+        ok(!run(app(['openssl', 'storeutl',
+                     '-subject', '/C=AU/ST=QLD/CN=SSLeay\/rsa test cert',
+                     srctop_file('test', 'testx509.pem')])),
+           "Checking that -subject can't be used with a single file");
```

Comment:
Why not?

Reply:
Because of limitations in the `file:` backend, that only supports searching by subject name when the URI represents a catalogue.  This is because the subject name is hashed and the search is done by matching file names.

A possibility for a single file URI would be to simply compare the name of the certs included in the file and only return those that match.  Is that desirable?

Reply:
> A possibility for a single file URI would be to simply compare the name of the certs included in the file and only return those that match. Is that desirable?

Sounds useful to me. But could be a different PR.